src/com/hillel/oRudoiPackage/oRudoiClass.java

```diff
@@ -0,0 +1,8 @@
+package com.hillel.oRudoiPackage;
+
+public class oRudoiClass {
+
```

Comment:
Please remove the empty line

Reply:
Is this just for practice or a working tip? Empty lines between logical blocks are useful for code style.

Reply:
Its for practice and working tip. Recommended but not required. Empty lines are useful between logical blocks but not before blocks